extensions/gnome/desktop-gnome-specific

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+##############################
+# GTK launcher specific part #
+##############################
+
+if [ "$SNAP_DESKTOP_WAYLAND_AVAILABLE" = "true" ]; then
+  export GDK_BACKEND="wayland"
+  export CLUTTER_BACKEND="wayland"
+  # Does not hurt to specify this as well, just in case
+  export QT_QPA_PLATFORM=wayland-egl
+  export GTK_PATH="$SNAP_DESKTOP_RUNTIME/usr/lib/$SNAP_LAUNCHER_ARCH_TRIPLET/gtk-3.0"
```

Comment:
This is not wayland-specific. Moving the export of `$GTK_PATH` outside of this block fixes ibus input in the libreoffice snap.